README.md

```diff
@@ -1,4 +1,10 @@
-<img alt="Gitpod Logo" src="https://raw.githubusercontent.com/gitpod-io/gitpod/master/components/dashboard/public/images/gitpod-ddd_light.svg" width="300">
+<p align="center">
+  <a href="https://www.gitpod.io">
+    <img src="https://raw.githubusercontent.com/gitpod-io/gitpod/master/components/dashboard/public/images/gitpod.svg" height="60">
+    <h3 align="center">Gitpod</h3>
+  </a>
+  <p align="center">Always ready-to-code.</p>
+</p>
```

Comment:
No strong opinion here, but I like the existing left alignment better. It somehow is easier on the eye/reading flow.

Reply:
Got inspired by https://github.com/vercel/vercel and think the centralized structure looks cleaner and gives us the option to display both 1) link to the website and 2) our claim.